Dockerfile.build

```diff
@@ -32,11 +32,11 @@ RUN set -eux; \
 	; \
 	rm -rf /var/lib/apt/lists/*
 
-# https://www.musl-libc.org/download.html
+# https://musl.libc.org/
```

Comment:
```suggestion
# https://musl.libc.org/releases.html
```

(Somehow, I forgot to add this comment....  sorry :facepalm:)

Reply:
Done. :-)